addons/im_livechat/static/src/models/thread/thread.js

```diff
@@ -20,6 +22,8 @@ registerClassPatchModel('mail.thread', 'im_livechat/static/src/models/thread/thr
         if ('livechat_visitor' in data && data.livechat_visitor) {
             if (!data2.members) {
                 data2.members = [];
+            } else if(data2.members instanceof FieldCommand) {
+                data2.members = [data2.members];
```

Comment:
it is needed because originally, we have a command like `[ [command1] ]`, and then it can be pushed with a new `[]` command to be like `[ [command_1], [command_2] ]`,  but now, instead of `[[]]` command, `members` in super returns `FieldCommand` Object.